src/bindings/capi/tvgCapi.cpp

```diff
@@ -194,7 +194,7 @@ TVG_EXPORT Tvg_Result tvg_paint_get_opacity(Tvg_Paint* paint, uint8_t* opacity)
 TVG_EXPORT Tvg_Result tvg_paint_get_bounds(const Tvg_Paint* paint, float* x, float* y, float* w, float* h)
```

Comment:
I think we need to replace this to  tvg_paint_get_bounds(const Tvg_Paint* paint, float* x, float* y, float* w, float* h, bool transformed); as same to c++ version.
Could you please make the patch? (separate patch is welcomed)

Reply:
I thought about it, the problem is that another library uses this API so such a change would break backwards compatibility. You would have to bump up the version, so I think a separate commit would be a better solution.

Reply:
Agree